### PR TITLE
Add github usernames besides projects on /projects route

### LINKED ISSUE #113: Project Produced By
**Context:**
Seen it a few times where people were asking who worked on what project and didn't know.

**Proposal:**

*Edit: On the `/projects` page (and any translated page `/es`, `/fr`, etc)

Option 1 - Add the github username:

```md
[Developer DAO](https://developerdao.vercel.app/) - by [fmoliveira](https://github.com/fmoliveira)
```

Option 2 - Add name when username not available:

```md
[Developer DAO](https://developerdao.vercel.app/) - by Filipe Oliveira
```

**Work Needed**
- [ ] Find there github usernames for the projects to link to them
- [ ] Modify the `projects` page
- [ ] Add translations for `by` in the common.json files (coordinate to get as many official languages as possible)

**Questions:**

👍 If you like this idea (we should do it)
👎 If you don't like this idea (aka don't do it)
🆕 If you have a different proposal

### PR DESCRIPTION
**Pull request by Nazeeh21**
### What does it do?
Add GitHub usernames of the project who worked on besides the project name on `/projects` route and add a translation for `by` in `en` language.

### Any helpful background information?
--

### Any new dependencies? Why were they added?
--

### Relevant screenshots/gifs
image

### Does it close any issues?

Closes #113

### CODE REVIEW

**Comment by Nazeeh21** [comment]
For now, I have added placeholder names besides the project. I don't know the project owners so I haven't added that. If you provide me their names, I'll update the PR with their names and GitHub links.

File: src/pages/projects.tsx [hunk 1]
```diff
@@ -24,12 +25,21 @@ function Projects() {
           </Text>
           <UnorderedList spacing="1">
             <ListItem>
-              <Link href="https://ddao.ibby.dev/">DDAO Token Search</Link>
+              <Flex alignItems="center">
+                <Link href="https://ddao.ibby.dev/">DDAO Token Search</Link>
+                <Box mx={1}>- {t('by')}</Box>
+                <Text>Filipe Oliveira</Text>
```
Comment (codingwithmanny) [inline src/pages/projects.tsx:31]: I think this is @Ibby-devv (discord: IbbyDev#8201)
Might need to confirm with them.

Comment (Nazeeh21) [reply src/pages/projects.tsx:31]: Okay! I'll update them as soon as @Ibby-devv approves it.

Comment (with-heart) [reply src/pages/projects.tsx:31]: I think you're good to make the change. Very confident that @ibby-dev is the correct profile.

File: src/pages/projects.tsx [hunk 2]
```diff
@@ -24,12 +25,21 @@ function Projects() {
...
-              <Link href="https://developerdao.vercel.app/ ">
-                Developer DAO
-              </Link>
+              <Flex alignItems="center">
+                <Link href="https://developerdao.vercel.app/ ">
+                  Developer DAO
+                </Link>
+                <Box mx={1}>- {t('by')}</Box>
+                <Link href="https://github.com/fmoliveira">fmoliveira</Link>
+              </Flex>
```
Comment (with-heart) [inline src/pages/projects.tsx:42]: Shouldn't need to use `Flex` here

```tsx
<ListItem>
  <Link>Developer DAO</Link> - {t('by')} <Link>fmoliviera</Link>
</ListItem>
```

If we don't use `Box` or `Text`, there's nothing forcing the lines to break

**Review by with-heart — CHANGES_REQUESTED** [review]
Looks good! Just one requested change to simplify the list elements structure

File: src/pages/projects.tsx (hunk 2 above)
Comment (Nazeeh21) [reply src/pages/projects.tsx]: I have made the changes and also updated the owner name for *DDAO Token Search*.

**Review by with-heart — APPROVED** [review]
I think we're good to go!

**Review by codingwithmanny — APPROVED** [review]
LGTM

**Pull request merged**